### PR TITLE
[17.0][MIG] crm_stage_probability: Migration to 17.0

### PR DESCRIPTION
**Pull request by MohamedOsman7**

### CODE REVIEW

File: crm_stage_probability/views/crm_lead.xml [hunk 1]
```diff
@@ -7,14 +7,14 @@
         <field name="arch" type="xml">
             <xpath
                 expr="//button[@name='action_set_automated_probability']"
-                position="before"
+                position="after"
```
Comment (CRogos) [inline crm_stage_probability/views/crm_lead.xml]: I think this should stay before and should look like this
image

File: crm_stage_probability/views/crm_lead.xml [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
             >
                 <field name="is_stage_probability" invisible="1" />
                 <button
                     class="ps-0 ps-md-2 btn btn-link"
```
Comment (CRogos) [inline crm_stage_probability/views/crm_lead.xml]: Could you check if this class fixes the layout.
`class="btn d-inline-block px-2 py-0 btn-link"`

**Review by CRogos — COMMENTED** [review]
I think we can squash these commits into the migration 14.0 commit:
image

**Review by CRogos — APPROVED** [review]

**Comment by MohamedOsman7** [comment]
@remi-filament could you please review?

**Review by remi-filament — CHANGES_REQUESTED** [review]
Hi @MohamedOsman7 thank you for porting this module.
I get a warning when installing : 
`WARNING odoo17 py.warnings: /opt/odoo/addons/base/models/ir_module.py:178: DeprecationWarning: XML declarations in HTML module descriptions are deprecated since Odoo 17, crm_stage_probability can just have a UTF8 description with not need for a declaration.`
I think you need to remove the xml tags (first line of index.html)

File: crm_stage_probability/static/description/index.html [hunk 3]
```diff
@@ -0,0 +1,440 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (remi-filament) [inline crm_stage_probability/static/description/index.html]:
```suggestion

```

**Review by remi-filament — CHANGES_REQUESTED** [review]

**Comment by MohamedOsman7** [comment]
@remi-filament done

**Review by remi-filament — APPROVED** [review]
Thanks @MohamedOsman7 LGTM

**Pull request merged**